swift/apple/FirezoneKit/Sources/FirezoneKit/Views/MenuBar.swift

```diff
@@ -271,6 +282,18 @@ public final class MenuBar: NSObject, ObservableObject {
     }
   }
 
+  @objc private func installSystemExtensionButtonTapped() {
+    Task {
+      model.store.installSystemExtension()
+    }
+  }
+
+  @objc private func grantVPNPermissionButtonTapped() {
```

Comment:
Since we track the sign in button state as a combination of the system extension state and VPN tunnel permission state, it made sense to allow the user to re-request VPN permission here as well.